package.xml

```diff
@@ -19,6 +19,7 @@
   <build_depend>moveit_core</build_depend>
   <build_depend>moveit_ros_planning</build_depend>
   <build_depend>arc_utilities</build_depend>
+  <build_depend>pybind11_catkin</build_depend>
```

Comment:
`pybind11_catkin` is an optional dependency, so remove from `<build_depend>`

Reply:
yea I'm on the fence about this, because if you want to build python bindings it's unclear how you're supposed to get pybind11_catkin, but with this anyone doing `rosdep install` will just get the right thing and not have to worry about it. I've changed my mind and made the bindings build by default.